config/initializers/sunspot.rb

```diff
@@ -0,0 +1,6 @@
+# ensure that sunpot is loaded
```

Comment:
We aren't doing this any more, are we? You got rid of all the extra fancy extensions?